test/e2e/service_to_service_test.go

```diff
@@ -220,8 +220,6 @@ func TestServiceToServiceCall(t *testing.T) {
 	for _, tc := range testCases {
 		helloworldURL := resources.Route.Status.URL.URL()
 		t.Run(tc.name, func(t *testing.T) {
-			cancel := logstream.Start(t)
```

Comment:
e.g. `logstream.Start(t)` is already called at L182.

Reply:
We want to call this in the respective tests as well though. If anything, we would remove the outer logstream call. I would keep it in this case though, as we do some setup work outside of specific `t.Run` calls.

Reply:
Ah! Thank you.

Sorry, I misunderstood about https://github.com/knative/serving/pull/5807#discussion_r334855662 and filed this. Closing this now.